### PR TITLE
Pcm 573

### PR DESCRIPTION
**Pull request by renujhamtani**
@pbathia @ardasgup 

Please review and merge.

This contains the following fixes.

1) Initial pull discsusion and case history fixes which spenser made and was not working fully.
2) Bomgar session details changes
3) Also fixes for the scss changes for case discussion which was causing the case discussion to be all in block letters and also was affecting PCM - 1109

### CODE REVIEW

File: app/ascension/services/caseDetailsService.js [hunk 1]
```diff
@@ -55,7 +56,8 @@ angular.module('RedhatAccess.ascension').service('CaseDetailsService', [
         this.getCaseDetails = function(caseNumber) {
             AccountService.accountDetailsLoading = true;
             this.caseDetailsLoading = true;
-            udsService.kase.details.get(caseNumber).then(angular.bind(this, function (response) {
+            var promise=udsService.kase.details.get(caseNumber);
```
Comment (ardasgup) [inline app/ascension/services/caseDetailsService.js]: We need to add a defer also, in case the fetch fails.

**Comment by ardasgup** [comment]
@renujhamtani rest looks good !

**Comment by renujhamtani** [comment]
@ardasgup Incorporated the review comments. please check

**Comment by ardasgup** [comment]
Reviewed!

**Comment by pbathia** [comment]
Reviewed.

**Pull request merged**